core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.html

```diff
@@ -4,7 +4,7 @@
      on-finish="$ctrl.onFinishTutorial()"
      on-skip="$ctrl.onSkipTutorial()">
   <div class="exp-editor-main-editor">
-    <div class="oppia-editor-cards-container">
+    <div class="oppia-editor-cards-container protractor-test-fade-in">
```

Comment:
Added a class `protractor-test-fade-in` to wait for the fade in process to complete.